.travis.yml

```diff
@@ -15,9 +15,18 @@ install:
 
 script:
   - ./gradlew build
-  - if [[ ${TRAVIS_PULL_REQUEST} == "false" ]]; then ./gradlew  artifactoryPublish -Partifactory_url=https://meremortal.jfrog.io/meremortal/ -Partifactory_user=${ARTIFACTORY_USER} -Partifactory_password=${ARTIFACTORY_PASS}; fi
   - |-
     docker build \
     --build-arg version="$(./gradlew -q printVersion)" \
     --build-arg war="$(./gradlew -q printWarName)" \
     -t crispy-barnacle:latest .
+  - |-
+    if [[ ${TRAVIS_PULL_REQUEST} == "false" ]]; then
+      ./gradlew  artifactoryPublish 
+      -Partifactory_url=https://meremortal.jfrog.io/meremortal/
+      -Partifactory_user=${ARTIFACTORY_USER}
+      -Partifactory_password=${ARTIFACTORY_PASS};
+      docker login -u ${ARTIFACTORY_USER} -p ${ARTIFACTORY_PASS} meremortal-mortal-docker.jfrog.io
+      docker tag crispy-barnacle meremortal-mortal-docker.jfrog.io/crispy-barnacle:latest
+      docker push meremortal-mortal-docker.jfrog.io/crispy-barnacle:latest
```

Comment:
Travis is wonky here, it will want semi-colons after every line in the `if` block.